Sources/TuistSigning/SigningInstaller.swift

```diff
@@ -1,37 +1,29 @@
 import Foundation
 import TSCBasic
+import TuistCore
 import TuistSupport
 
-enum SigningInstallerError: FatalError, Equatable {
-    case noFileExtension(AbsolutePath)
-    case provisioningProfilePathNotFound(ProvisioningProfile)
-    case revokedCertificate(Certificate)
-    case expiredProvisioningProfile(ProvisioningProfile)
-
-    var type: ErrorType {
-        switch self {
-        case .noFileExtension, .provisioningProfilePathNotFound, .revokedCertificate, .expiredProvisioningProfile:
-            return .abort
-        }
+extension LintingIssue {
```

Comment:
Since `.noFileExtension` is not tied to `SigningInstaller` I'd recommend moving it to the `LintingIssue` implementation without using extensions.

Reply:
are you sure? right now it's only used by `SigningInstaller` (and I'm not even sure why it's needed tbh).
I think I'd rather keep it here for now and move it when it's used somewhere else.